openfisca/api_config.ini

```diff
@@ -0,0 +1,49 @@
+# OpenFisca-Web-API - Development environment configuration
+#
+# The %(here)s variable will be replaced with the parent directory of this file.
+
+[DEFAULT]
+debug = true
```

Comment:
If set to `false`, openfisca will try to send emails when an error happens. We never used it before, we could try, but there may be heavy configuration to be able to send emails.